task_02/index.html

```diff
@@ -1,11 +1,195 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    Task 2
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Desktop navigation -->
+      <nav role="navigation" class="desktop-navigation">
+        <div class="nav-wrapper">
+          <div class="list-item"><a href="" class="links">Home</a></div>
+          <div class="list-item"><a href="" class="links">Approach</a></div>
+          <div class="list-item"><a href="" class="links">Work</a></div>
+          <div class="list-item"><a href="" class="links">People</a></div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Jobs</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Blog</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Contact</a>
+          </div>
+        </div>
+      </nav>
+
+      <!-- Mobile nav -->
+      <nav role="navigation" class="mobile-navigation">
+        <div id="menuToggle">
+          <!--
+        A fake / hidden checkbox is used as click reciever,
+        so you can use the :checked selector on it.
+        -->
+          <input type="checkbox" id="toggleCheckbox" onclick="moveText()" />
+
+          <!--
+        Some spans to act as a hamburger.
+        
+        They are acting like a real hamburger,
+        not that McDonalds stuff.
+        -->
+          <span></span>
+          <span></span>
+          <span></span>
+
+          <!--
+        Too bad the menu has to be inside of the button
+        but hey, it's pure CSS magic.
+        -->
+          <ul id="menu">
+            <a href="#"><li>Home</li></a>
+            <a href="#"><li>Approach</li></a>
+            <a href="#"><li>Work</li></a>
+            <a href="#"><li>People</li></a>
+            <a href="#" id="disabled"><li>Jobs</li></a>
+            <a href="#" id="disabled"><li>Blog</li></a>
+            <a href="#" id="disabled"><li>Contact</li></a>
```

Comment:
Same for ID, it has to be unique

Reply:
It can be class also,it is just css effect of disabled items.